extensions/bootstrap/Dropdown.php

```diff
@@ -15,6 +15,21 @@
 /**
  * Dropdown renders a Bootstrap dropdown menu component.
  *
+ * For example,
+ *
+ * ```html+php
```

Comment:
I updated syntax highlighting scheme.
BTW: github uses Linguist to perform language detection and [syntax highlighting](https://help.github.com/articles/github-flavored-markdown/#syntax-highlighting). Linguist supports `html+php` so I think github also supports this scheme.